language-reference-manual/lrm.trlc

```diff
@@ -1188,10 +1188,6 @@ section "Names" {
               the given integer to the exact same rational.'''
   }
 
-  Name_Resolution Case_Sensitive {
```

Comment:
Agreed to remove this requirement. Justification:
- Since TRLC v 2.0.0, names must be non-similar. For example, `name` and `nAmE` are too similar and not allowed. A requirement to have case sensitive name resolution is hence misleading.
- Built-in functions are only resolved, if their names are matched exactly. For example, no function called `LEN` exists, and the user must write `len` instead. There is a separate requirement explicitly targeting built-in function names. Again, no need for the current requirement.
- It is unclear how to test this requirement, as names exists in the following four contexts (see LRM). It is unclear whether this requirement shall apply for all of them, and where to find the implementations of the case sensitivity in the code.

In total, I agree it is better to remove this requirement.